File: java/strings/RemoveCharacter.java
public class RemoveCharacter {
	
	public static void print(Object words) {
		System.out.println(words);
	}

	public static void main(String[] args) {
		print("Original: " + args[1]);
		print("Reversed: " + args[1].replace(args[0], "") + "\n");
	}
}

File: python/strings/remove-character.py
import sys

def remove_character(word, character):
        return word.replace(character, "")


words = sys.argv

character = words[1]

print "\nCharacter to be Removed: ", character

for count, word in enumerate(words):
    if count != 0 and count != 1:
        print "Original String: ", word
        print "Removed Character: ", remove_character(word, character)
    count += 1